packages/esm-active-visits-app/src/home-page-tiles/metrics-slot.component.tsx

```diff
@@ -0,0 +1,13 @@
+import { ExtensionSlot } from '@openmrs/esm-framework';
+import React from 'react';
+
+const MetricsSlot = () => {
```

Comment:
This becomes tricky when we're adding tiles from other esms. Can we maintain this in the esm home?

Reply:
My though process behind this:
- We need a way to manage the parent container of the metric tiles that does not rely on `openmrs-esm-home` since that slot should be totally generic and agnostic of whatever is loaded into it.
- This works since the different tiles reside within different esms (`esm-active-visits-app`, `esm-appointments-app`) but need to live inside a single container for the purposes of uniformity of styling/ alignment, and also so that they are loaded into the extension slot within `openmrs-esm-home` as a single unit instead of three individual tiles.

Reply:
This is my understanding;
- This component(`MetricSlot`) introduces the Metrics slot which is specific to the home. 
- Different esm can then define tiles that are attached to this slot.

What am suggesting is this specific component defining the metrics-slot should be `esm-home` and `patient-management` let alone the `esm-visits-app`. It's not a tied to the visits esm and implementers who decide not to include the `esm-visits` in the importmap would have this. But implementations that have `esm-home` should be able to take advantage of the `metrics-tiles-slot`

Reply:
Aah, I see what you mean! Which begs the question, where within `patient-management` would you suggest I define this? Could there be a way to represent this logic independent of any of the modules? I hope my understanding of what you said was correct.

Reply:
The slot would move to the `esm-home` and the different tiles would be attached in the routes.json of their respective esms